src/libraries/System.Net.Connections/ref/System.Net.Connections.cs

```diff
@@ -71,6 +71,7 @@ public partial interface IConnectionProperties
     {
         bool TryGet(System.Type propertyKey, [System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] out object? property);
     }
+    [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

Comment:
This whole assembly is unsupported on browser (and beyond that is currently dead code... we shouldn't be spending any time maintaining it right now). cc: @geoffkizer

Reply:
> (and beyond that is currently dead code... we shouldn't be spending any time maintaining it right now)

Got it thanks, i still need to suppress the warning, so i guess i leave it as is

Reply:
> so i guess i leave it as is

Wouldn't it be better to just mark the whole assembly?

Reply:
> Wouldn't it be better to just mark the whole assembly?

Sure i can do that

> we shouldn't be spending any time maintaining it right now

From this, i thought you are saying we shouldn't do anything 😛

Reply:
> From this, i thought you are saying we shouldn't do anything

I'm fine with us doing nothing or marking the whole assembly.  I thought maybe mistakenly that your "leave it as is" meant as you had it in the pr, which is what I was pushing back on.